pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.2</version>
+    <version>1.580.1</version>
```

Comment:
My preference is to keep the baseline until forced to update as that way *if* anyone wants to cherry pick backports it is easier